src/PrestaShopBundle/Twig/HookExtension.php

```diff
@@ -134,7 +134,7 @@ public function renderHooksArray($hookName, $hookParameters = array())
             $render[] = [
                 'id' => $module,
                 'name' => $this->moduleDataProvider->getModuleName($module),
-                'content' => array_values($hookRender)[0],
+                'content' => $hookRender,
```

Comment:
it will break another bug fixed here => https://github.com/PrestaShop/PrestaShop/pull/10038

Revert some update introduced is never a good idea, you need to be more precise and/or add a specific test

Reply:
hi again @dheerajwebkul 
so after some additional tests we validate your changes, however could you change your PR and target the 1.7.5.x branch instead (the change will be merged back in develop later)
I also would like you to integrate one small fix in https://github.com/PrestaShop/PrestaShop/blob/e078125d0700056c0579f28b0e4c1a0aaf4ab66a/src/PrestaShopBundle/Service/Hook/RenderingHookEvent.php#L84

and change

```php
    $this->currentContent = '';
```

with

```php
    $this->currentContent = [];
```